packages/core/src/TolgeeConfig.ts

```diff
@@ -74,7 +74,7 @@ export class TolgeeConfig {
     | (keyof HTMLElementTagNameMap)[]
     | ((node: Element) => boolean) = ['option', 'optgroup'];
   restrictedElements?: string[] = ['script', 'style'];
-  highlightColor?: string = 'rgb(224 240 255)';
+  highlightColor?: string = 'rgb(255, 0, 0)';
```

Comment:
I was thinking about consulting the styles of the highlighted element with Zuzka. Maybe it would be nice if it was in Tolgee purple or similar. 

Also guys from Homez suggested custom styling of the highlighting element, which is now possible via the class, isn't it?